### PR TITLE
LPS-135809 Fix test ViewIdPSingleLogoutWithMultipleSPs

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-135809

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -1281,57 +1274,81 @@ definition {
...
 		SAML.configureLiferaySAMLAsSP(
 			samlEntityID = "samlsp",
-			spURL = "http://www.able.com:9080");
+			spAttributes = '''emailAddress
+				firstName
+				lastName
+				screenName
+				uuid''',
+			spURL = "http://www.able.com:8080",
+			userEmailAddress = "test@www.able.com",
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1329]: @weller0246, According to the test's original design, at this point, what will be the Instance admin's email address?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 2]
```diff
@@ -1281,57 +1274,81 @@ definition {
...
 		SAML.configureLiferaySAMLAsSP(
 			samlEntityID = "samlsp2",
-			spURL = "http://www.baker.com:9080");
+			spAttributes = '''emailAddress
+				firstName
+				lastName
+				screenName
+				uuid''',
+			spURL = "http://www.baker.com:8080",
+			userEmailAddress = "test@www.baker.com",
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1342]: @weller0246, According to the test's original design, at this point, what will be the Instance admin's email address?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 3]
```diff
@@ -1344,27 +1361,25 @@ definition {
...
 		// Verify second SP is prompted to be logged in after click sign in button
 
-		Navigator.openSpecificURL(url = "http://www.baker.com:9080");
+		Navigator.openSpecificURL(url = "http://www.baker.com:8080");
 
 		Navigator.gotoLoginPage();
 
-		User.viewLoggedInPG();
-
-		// Initiate Single Logout for IdP
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1353]: @weller0246, Please leave the comments which help the test's readability.

File: test.properties [hunk 4]
```diff
@@ -4593,7 +4593,7 @@
     test.batch.names[saml]=${test.batch.names[security]}
 
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][saml]=\
-        (portal.upstream == true) AND \
+        (portal.upstream == "marci") AND \
```
Comment (gaborlovasdotcom) [inline test.properties:4596]: Shall be removed before forwarding.

File: test.properties [hunk 5]
```diff
@@ -4742,14 +4742,7 @@
...
     test.batch.names[security]=\
-        central-requirements-jdk8,\
-        functional-tomcat90-mysql57-jdk8,\
-        js-unit-jdk8,\
-        modules-integration-mysql57-jdk8,\
-        modules-semantic-versioning-jdk8,\
-        modules-unit-jdk8,\
-        semantic-versioning-jdk8,\
-        unit-jdk8
+        functional-tomcat90-mysql57-jdk8
```
Comment (gaborlovasdotcom) [inline test.properties:4745]: Shall be removed before forwarding.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please fix the test, test out locally, and update the successful test result to the ticket.

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

**Review by ferenconodiD — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 6]
```diff
@@ -1281,57 +1274,81 @@ definition {
 			virtualHost = "www.baker.com",
 			webId = "www.baker.com");
 
+		MyAccount.openMyAccountAdmin(baseURL = "http://www.test.com:8080");
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:1277]: @weller0246 Could you please change this URL to http://localhost:8080, according to the test case terminolodgy.

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Review by gaborlovasdotcom — APPROVED** [review]

**Comment by gaborlovasdotcom** [comment]
Test case [passed](https://testray.liferay.com/home/-/testray/case_results/1548870789?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1548870716%26cur%3D1%26delta%3D200%26tab%3Druns%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc) in test run

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**